mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensorAttrDefs.td

```diff
@@ -167,7 +167,7 @@ def SparseTensorEncodingAttr : SparseTensor_Attr<"SparseTensorEncoding",
     - **soa** : only applicable to singleton levels, fuses the singleton
       level in SoA (structure of arrays) scheme.
 
-    In addition to the map, the following two fields are optional:
+    In addition to the map, the following four fields are optional:
```

Comment:
maybe just say, the following fields are optional
so we don't have to change that each time